sandbox/src/filters.rs

```diff
@@ -0,0 +1,53 @@
+use slog::Logger;
+use warp::Filter;
+
+use crate::handlers::{start_node_with_config, stop_node};
+use crate::node_runner::LightNodeRunnerRef;
+
+pub fn sandbox(
+    log: Logger,
+    runner: LightNodeRunnerRef,
+) -> impl Filter<Extract = impl warp::Reply, Error = warp::Rejection> + Clone {
+    start(log.clone(), runner.clone()).or(stop(log.clone(), runner))
```

Comment:
@adonagy 
I think, you dont need to clone second logger `stop(log.clone(), runner)` -> `stop(log, runner)`